### PR TITLE
[en] Clean up various broken links

### PR DESCRIPTION
**Pull request by tallclair**
Clean up broken links reported through the HackerOne bug bounty program.

Localized version of #32783

/committee security-response

### CODE REVIEW

File: content/en/blog/_posts/2016-04-00-Kubernetes-Network-Policy-APIs.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ In this example, the ‘ **tenant-a** ’ namespace would get policy ‘ **pol1*
...
+Today, [Romana](https://github.com/romana/romana), [OpenShift](https://www.openshift.com/), [OpenContrail](http://www.opencontrail.org/) and [Calico](http://projectcalico.org/) support network policies applied to namespaces and pods. Cisco and VMware are working on implementations as well. Both Romana and Calico demonstrated these capabilities with Kubernetes 1.2 recently at KubeCon. You can watch their presentations here: [Romana](https://www.youtube.com/watch?v=f-dLKtK6qCs) ([slides](http://www.slideshare.net/RomanaProject/kubecon-london-2016-ronana-cloud-native-sdn)), [Calico](https://www.youtube.com/watch?v=p1zfh4N4SX0) ([slides](http://www.slideshare.net/kubecon/kubecon-eu-2016-secure-cloudnative-networking-with-project-calico)).&nbsp;
```
Comment (sftim) [inline content/en/blog/_posts/2016-04-00-Kubernetes-Network-Policy-APIs.md:134]: SIG Docs policy is not to revise blog posts more than 1 year old. We add a banner advising readers to check pages haven't gone out of date.

We prefer not to make partial fixes; I'd like to also remove the broken link to OpenContrail.

```suggestion
Today, Romana, OpenShift, OpenContrail and Calico support network policies applied to namespaces and pods. Cisco and VMware are working on implementations as well. Both Romana and Calico demonstrated these capabilities with Kubernetes 1.2 recently at KubeCon. You can watch their presentations here: [Romana](https://www.youtube.com/watch?v=f-dLKtK6qCs) ([slides](http://www.slideshare.net/RomanaProject/kubecon-london-2016-ronana-cloud-native-sdn)), [Calico](https://www.youtube.com/watch?v=p1zfh4N4SX0) ([slides](http://www.slideshare.net/kubecon/kubecon-eu-2016-secure-cloudnative-networking-with-project-calico)).
```

File: content/en/blog/_posts/2016-09-00-High-Performance-Network-Policies-Kubernetes.md [hunk 2]
```diff
@@ -189,4 +189,4 @@ These tests were performed using Romana as the backend policy provider and other
 
 
 
-If you wish to try it for yourself, we invite you to check out [Romana](http://romana.io/). In our [GitHub repo](https://github.com/romana/romana) you can find an easy to use installer, which works with AWS, Vagrant VMs or any other servers. You can use it to quickly get you started with a Romana powered Kubernetes or OpenStack cluster.
+If you wish to try it for yourself, we invite you to check out [Romana](https://github.com/romana/romana). In our [GitHub repo](https://github.com/romana/romana) you can find an easy to use installer, which works with AWS, Vagrant VMs or any other servers. You can use it to quickly get you started with a Romana powered Kubernetes or OpenStack cluster.
```
Comment (sftim) [inline content/en/blog/_posts/2016-09-00-High-Performance-Network-Policies-Kubernetes.md:192]:
```suggestion
If you wish to try it for yourself, we invite you to check out Romana. In our [GitHub repo](https://github.com/romana/romana) you can find an easy to use installer, which works with AWS, Vagrant VMs or any other servers. You can use it to quickly get you started with a Romana powered Kubernetes or OpenStack cluster.
```

**Review by sftim — COMMENTED** [review]
SIG Docs policy is not to revise blog posts more than 1 year old. We add a banner advising readers to check pages haven't gone out of date.

http://romana.io/ is not malware, it's just a broken link.

I want to record that accepting this change does not set a precedent for other PRs that propose to revise old articles.

How do these suggestions look?

**Comment by tallclair** [comment]
> http://romana.io/ is not malware, it's just a broken link.

The security risk is that someone could purchase that domain, make a look-alike page to the real romana destination, and provide malware downloads. I agree it's low priority, but that's why we're treating this as a security vulnerability. Note that this only applies to nxdomains (or unregistered domains), not to broken paths that 404.

**Comment by sftim** [comment]
Usual SIG Docs blog policy is not to revise blog posts more than 1 year old; we can make an exception of course.

How do you feel about _removing_ the links to Romana as I've suggested @tallclair ?

**Comment by tallclair** [comment]
Yeah, I'm fine with removing the links.

File: content/en/blog/_posts/2017-08-00-High-Performance-Networking-With-Ec2.md [hunk 3]
```diff
@@ -73,8 +73,5 @@ When using Romana v2.0, native VPC networking is now available for clusters of a
 ![](https://archive.org/download/hpc-ec2-vpc-2/hpc-ec2-vpc-2.png)
 
 
-The preview release of Romana v2.0 is available [here](http://romana.io/preview). We welcome comments and feedback so we can make EC2 deployments of Kubernetes as fast and reliable as possible.
```
Comment (nate-double-u) [inline content/en/blog/_posts/2017-08-00-High-Performance-Networking-With-Ec2.md:76]: I'd prefer not to remove this whole paragraph. Could we do something like this instead:

```suggestion
The preview release of Romana v2.0 is available here _Editor's Note:_ link removed. We welcome comments and feedback so we can make EC2 deployments of Kubernetes as fast and reliable as possible.    
```

@sftim do we have any history of providing Editor's Notes? is this the format they'd be?

Comment (zacharysarah) [reply content/en/blog/_posts/2017-08-00-High-Performance-Networking-With-Ec2.md:76]: @nate-double-u We've used editor's notes at the top of other blog posts. I don't like the format of this one, but I'd rather expedite the security concern so we can deliberate formatting in a non-blocking way.

**Comment by zacharysarah** [comment]
@tallclair Thanks for your patience.

/lgtm
/approve

**Pull request merged**